frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -116,6 +123,37 @@ export const surveyLogic = kea<surveyLogicType>([
                 return await api.surveys.update(props.id, { end_date: null })
             },
         },
+        surveyUserStats: {
+            loadSurveyUserStats: async (): Promise<SurveyUserStats> => {
+                const query: HogQLQuery = {
+                    kind: NodeKind.HogQLQuery,
+                    query: hogql`
+                        SELECT
+                            (SELECT COUNT(DISTINCT person_id)
+                                FROM events
+                                WHERE event = 'survey shown' AND properties.$survey_id = ${props.id}
+                                AND person_id NOT IN (
```

Comment:
I think you can do this subtraction in javascript once you get the three values, will be much faster :) - unless I'm missing a case where this count is different? 🤔 

Also, add the timestamp filters like in `surveyMetricsQuery` :)

Otherwise, looking good!